### DISCUSSION TITLE
Possible to have full Profile Card when clicking on persona card?

### DISCUSSION THREAD

**Discussion by mariaarcena** [microsoft-search/pnp-modern-search#1748] [Q&A] [answered]
Is it possible to have the full Profile Card when clicking on the persona card (below) when results are in People layout?

Currently when we have results to display through People layout, we the contact/people card when the mouse hovers on a persona card. Is there a simple/complex configuration that I am missing that can achieve the above?

(let me know if I should change the category to something else!)

Profile Card
![image](https://user-images.githubusercontent.com/17426994/150872051-ecab6a9e-8cd4-41d2-8103-fc5c5e85f60d.png)

**Comment by wobba** [comment, marked as answer]
Duplicate of #82 and #801. It's a current technical limitation as there are no official API's to invoke the persona card.

**Reply by mariaarcena** [reply]
Oh okay! I understand :) Thank you for pointing these out! 😊
